Backpack/src/main/java/net/skyscanner/backpack/starrating/internal/BpkStarRatingBase.kt

```diff
@@ -85,6 +97,7 @@ open class BpkStarRatingBase internal constructor(
     ).use {
       _maxRating = it.getInt(R.styleable.BpkStarRating_maxRating, maxRating)
       _rating = it.getFloat(R.styleable.BpkStarRating_rating, maxRating / 2f)
+      _rounding = it.getInt(R.styleable.BpkStarRating_rounding, rounding.ordinal).let { id -> RoundingType.values()[id] }
```

Comment:
We don't usually rely on ordinals and use int constants instead – this gives us more flexiblity when modifying enums. You can see an example in BpkBadge, for instance.